internal/crossdock/client/tchclient/thrift.go

```diff
@@ -50,6 +50,8 @@ func runThrift(t crossdock.T, call call) {
 	if checks.NoError(err, "thrift: call failed") {
 		assert.Equal(token, resp.Boop, "body echoed")
 		internal.RemoveVariableMapKeys(respHeaders)
+		// we manually remove service name header. Handler use YARPC, but client call is issued by raw tchannel client
```

Comment:
Thanks a lot, Alex. I really appreciated these comments since I also get english training during my internship, but I will try to avoid these trivial errors next time:)